src/lib/units/ph.ts

```diff
@@ -33,5 +33,5 @@ export class PH {
     this.value = value;
   }
 
-  getValue = () => Math.round((this.value * 1E4 / 1.5625) * ROUND_FACTOR) / ROUND_FACTOR;
```

Comment:
I believe this was a bug caught when implementing tests. this conversion comes from conductivity ppm conversion